### ISSUE TITLE
Getting private services from container is deprecated

### ISSUE THREAD

**Issue by javiereguiluz** [whiteoctober/WhiteOctoberPagerfantaBundle#174]
Services that need to public need to explicitly marked as public starting from Symfony 3.4. See http://symfony.com/blog/new-in-symfony-3-4-services-are-private-by-default.

Deprecations spotted while running tests in an app upgraded to Symfony 3.4:

```
The "white_october_pagerfanta.view_factory" service is private, getting it from
the container is deprecated since Symfony 3.2 and will fail in 4.0. You should
either make the service public, or stop using the container directly and use
dependency injection instead.
```

**Comment by yceruto** [comment]
@sampart please, see https://github.com/whiteoctober/WhiteOctoberPagerfantaBundle/pull/173

**Comment by sampart** [comment]
Ah, fab, thanks.  Sorry I missed that.

**Comment by sampart** [comment]
Fixed by #173
